docs/zope4/migration/zodb.rst

```diff
@@ -152,7 +173,7 @@ Further reading
 ~~~~~~~~~~~~~~~
 
 This guide is adapted from several sources that contain further information
-and examples.
+and examples. Be aware those sources may be outdated.
 
 * https://blog.gocept.com/2018/06/07/migrate-a-zope-zodb-data-fs-to-python-3/
```

Comment:
This one is outdated, I think it should marked so or even be removed.

Reply:
I removed those outdated links and added a link to the plone docs